<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Menu</class>
 <widget class="QMainWindow" name="Menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>548</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Main</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>15</number>
    </property>
    <property name="topMargin">
     <number>15</number>
    </property>
    <property name="rightMargin">
     <number>15</number>
    </property>
    <property name="bottomMargin">
     <number>15</number>
    </property>
    <item>
     <widget class="QPushButton" name="toSearchButton">
      <property name="text">
       <string>Search</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="toResistButton">
      <property name="text">
       <string>Resistance</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="toSignalButton">
      <property name="text">
       <string>Signal</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="toEmBipolarButton">
      <property name="text">
       <string>Emotions bipolar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="toEmMonopolarButton">
      <property name="text">
       <string>Emotions monopolar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="toSpectrumButton">
      <property name="text">
       <string>Spectrum</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="toChessboardButton">
      <property name="text">
       <string>Chessboard</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="disconnectButton">
      <property name="text">
       <string>Disconnect</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>366</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
